docs/content/en/_index.html

```diff
@@ -4,150 +4,826 @@
+            </div>
+            <ul class="tools__list">
+                <li class="tools__item">
+                    <a class="tools__link" href="https://labs.iximiuz.com/tutorials/introduction-to-tetragon" target="_blank" rel="noopener noreferrer">
+                        <img src="/svgs/promo-lab.svg" width="64" height="64" loading="lazy" alt="Test tube">
+                        <h3 class="tools__item-title">Tetragon lab</h3>
+                        <p class="tools__item-description">Try out the Introduction to Tetragon lab</p>
+                        <span class="button button--text">
+                            Try now
+                            <svg width="13" height="12" viewBox="0 0 13 12" fill="none" xmlns="http://www.w3.org/2000/svg"
+                                 aria-hidden="true">
+                                <path d="M1 5.12939C0.585787 5.12939 0.25 5.46518 0.25 5.87939C0.25 6.29361 0.585787 6.62939 1 6.62939V5.12939ZM12 5.12939L1 5.12939V6.62939L12 6.62939V5.12939Z"
+                                      fill="currentColor"/>
+                                <path d="M7.11108 1L12 5.7735L7.11108 10.7444" stroke="currentColor" stroke-width="1.5"
+                                      stroke-linecap="round" stroke-linejoin="round"/>
+                            </svg>
+                        </span>
+                    </a>
+                </li>
+                <li class="tools__item">
+                    <a class="tools__link" href="/docs/getting-started/">
+                        <img src="/svgs/promo-astronomy.svg" width="64" height="64" loading="lazy" alt="telescope"/>
+                        <h3 class="tools__item-title">Exploring Tetragon</h3>
+                        <p class="tools__item-description">A Security Observability Tool for Kubernetes, Docker, and
+                            Linux</p>
+                        <span class="button button--text">
+                            Try now
+                            <svg width="13" height="12" viewBox="0 0 13 12" fill="none" xmlns="http://www.w3.org/2000/svg"
+                                 aria-hidden="true">
+                                <path d="M1 5.12939C0.585787 5.12939 0.25 5.46518 0.25 5.87939C0.25 6.29361 0.585787 6.62939 1 6.62939V5.12939ZM12 5.12939L1 5.12939V6.62939L12 6.62939V5.12939Z"
+                                      fill="currentColor"/>
+                                <path d="M7.11108 1L12 5.7735L7.11108 10.7444" stroke="currentColor" stroke-width="1.5"
+                                      stroke-linecap="round" stroke-linejoin="round"/>
+                            </svg>
+                        </span>
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="videos">
+        <h2 class="title videos__title">
+            Watch videos on Tetragon technology
+            <a class="button button--text" href=/docs/resources#conference-talks">
```

Comment:
When I click in the preview it opens the URL `https://deploy-preview-2374--tetragon.netlify.app/docs/resources/#conference-talks%22` with the `"` symbol at the end, thus the anchor does not work, do you know why? can you fix this?